<?xml version="1.0" ?>

<launch>    
    <arg name="use_imu" default="true"/>
    <arg name="gazebo" default="false" />
    
    <param name="gazebo"                   value="$(arg gazebo)"    type="bool"/>
    <param name="gazebo_robot_name"        value="diana_soccer"/>
    
    <param name="offset_file_path"         value="$(find diana_manager)/config/offset.yaml"/>
    <param name="robot_file_path"          value="$(find diana_manager)/config/DIANA_SOCCER.robot"/>
    <param name="init_file_path"           value="$(find diana_manager)/config/dxl_init.yaml"/>
        
    <!-- <param name="action_file_path"         value="$(find thormang3_action_module)/data/motion_4096.bin"/> -->
                	 
    <!-- imu sensor package -->
    <!-- <include file="$(find thormang3_imu_3dm_gx4)/launch/imu.launch" if="$(arg use_imu)"/> --> 
    
    <!-- THORMANG3 Manager -->
    <node name="diana_soccer_manager" pkg="diana_manager" type="diana_soccer_manager" output="screen"/>
    
    <!-- Robot Model & TF -->
    <include file="$(find diana_description)/launch/diana_soccer_motion.launch"/> 
</launch>
